packages/studio/vite.config.ts

```diff
@@ -27,7 +28,12 @@ export default defineConfig((args: ConfigEnv): UserConfig => {
         input: ["index.html", "src/store/useStudioStore.ts"],
       },
     },
-    plugins: [react(), createStudioPlugin(args), svgr() as PluginOption],
+    plugins: [
+      react(),
+      createStudioPlugin(args),
+      createStudioStylingPlugin(),
+      svgr() as PluginOption,
+    ],
```

Comment:
out of curiosity, does the order of the plugins matter?

Reply:
I think in general, plugin order matters for Rollup and Vite, but in this case it works either before or after our `createStudioPlugin`.  We also `enforce` that this plugin is run as `pre`, which is before our `createStudioPlugin` which is unenforced (meaning that it runs after `pre` but before `post`.

Reply:
If we do want this plugin to run before `createStudioPlugin`, it may be safer to switch the order in case we ever end up adding `pre` to `createStudioPlugin`

Reply:
true, switched!